src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageServerClient.cs

```diff
@@ -96,43 +96,5 @@ public Task OnServerInitializedAsync()
         }
 
         public Task AttachForCustomMessageAsync(JsonRpc rpc) => Task.CompletedTask;
-
-        private class AutoFlushingStream : Stream
-        {
-            private readonly Stream _inner;
-
-            public AutoFlushingStream(Stream inner)
-            {
-                _inner = inner;
-            }
-
-            public override bool CanRead => _inner.CanRead;
-
-            public override bool CanSeek => _inner.CanSeek;
-
-            public override bool CanWrite => _inner.CanWrite;
-
-            public override long Length => _inner.Length;
-
-            public override long Position { get => _inner.Position; set => _inner.Position = value; }
-
-            // We intentionally call FlushAsync then don't await the task because calling Flush syncronously causes
-            // exceptions that can crash the extension.
```

Comment:
Does whatever this comment says no longer happen now that we are making this synchronous? Again, sorry if I missed an earlier conversation about this.

Reply:
From my understanding, we initially shifted from `_inner.Flush()` to `_inner.FlushAsync()` to avoid this issue. However, it seems it didn't solve the underlying issue as we were still seeing exceptions / crashes due to concurrent stream IO. I can add in a new comment to address the our reasoning for *synchronous* calls to async methods using `GetAwaiter().GetResult()`